src/lambda_function/s3_to_glue/template.yaml

```diff
@@ -27,13 +27,13 @@ Parameters:
 
   LambdaBatchSize:
     Type: Number
-    Default: 10
+    Default: 1
     Description: >-
       The maximum number of S3 messages in a SQS event that Lambda will process in a batch
 
   LambdaMaximumBatchingWindowInSeconds:
     Type: Number
-    Default: 300
+    Default: 60
```

Comment:
Yeah it seems like we really won't need this anymore since our S3 to JSON job has a maximum concurrency of ~150 to begin with.

Reply:
Nope, it doesn't make much sense to set it at all. But it doesn't hurt to keep it explicitly defined in the template, either.